pcdsdevices/lxe.py

```diff
@@ -215,6 +216,26 @@ def wm(self) -> float:
         return super().wm[0]
 
 
+class _ScaledUnitConversionDerivedSignal(UnitConversionDerivedSignal):
```

Comment:
If it's to be specific for this application, maybe it could be named `_NegativeUnitConversionDerivedSignal` or `_LXTUnitConversionDerivedSignal`?
Or if it's meant to be more generic (maybe it could be expanded to accept scale as an init parameter in a future PR), maybe remove the `_` and move to signal.py?

Reply:
🤷‍♂️ I think keeping the leading underscore `_` leaves room for changing it in the future. I'll change it if you feel strongly enough.

Reply:
True, keeping it private allows us to keep it as a WIP